README.md

```diff
@@ -21,12 +22,38 @@ in the course and create one group for each section and add all the users in eac
     * To download modules to default users home directory as part of Python system path just do `pip install --user -r requirements.txt`. 
 4. Add appropriate properties to the `config.yaml` and `security.yaml` as show in the template under `config/config.yaml`, `config/security.yaml` and could be placed out side of source code.
```

Comment:
I am just telling the add valid properties that are present in those file like `course_id`, `canvas_token` etc when running locally without openshift